src/main/java/hudson/remoting/ProxyOutputStream.java

```diff
@@ -341,7 +341,7 @@ public Unexport(int ioId, int oid) {
         protected void execute(final Channel channel) {
             channel.pipeWriter.submit(ioId,new Runnable() {
                 public void run() {
-                    channel.unexport(oid,createdAt);
+                    channel.unexport(oid,createdAt, false);
```

Comment:
minor comment: it seems like the standard is not to leave a blank after the comma. As I see in the arguments of the _submit_ method and in the old arguments of the unexport method. Do you consider removing the blank before the _false_?